source/sdk/CharBuffer.ooc

```diff
@@ -63,7 +63,8 @@ CharBuffer: class extends Iterable<Char> {
 			// if we were coming from a string literal, copy the original data as well (gc_realloc cant work on text segment)
 			if (size > 0 && mallocAddr == null) {
 				rs = 0
-				memcpy(tmp, data, size)
+				if (tmp != null && data != null)
```

Comment:
Yes, this is on purpose. I think coverity engine doesn't understand that this line can be never reached because exception is thrown.